### PR TITLE
Initial commit for new blog post 05-04-15

### PR DESCRIPTION
**Pull request by amaladolcevita**

### CODE REVIEW

**Comment by abtris** [comment]
Please upgrade first master to latest version before create new branch. You need rebase this branch with master to merge this PR in the future.

File: _config.yml [hunk 1]
```diff
@@ -75,11 +75,19 @@ authors:
     display_name: František
     email: "frantisek@apiary.io"
     twitter: "@Baggz"
+<<<<<<< HEAD
```
Comment (abtris) [inline _config.yml]: You have to remove this line.

File: _config.yml [hunk 2]
```diff
@@ -75,11 +75,19 @@ authors:
...
+  dave:
+    name: Dave Goldberg
+    display_name: Dave
+    email: "dave.goldberg@capitalone.com"
+    twitter: "@davidgoldberg"
+=======
```
Comment (abtris) [inline _config.yml]: You have to remove this line.

File: _config.yml [hunk 3]
```diff
@@ -75,11 +75,19 @@ authors:
...
   vincenzo:
     name: Vincenzo Chianese
     display_name: Vincenzo
     email: "vincenzo@apiary.io"
     twitter: "@D3DVincent"
+>>>>>>> master
```
Comment (abtris) [inline _config.yml]: You have to remove this line.

**Comment by smizell** [comment]
Closing and using #50 instead

**Pull request closed without merging**